magenta/interfaces/midi/midi_hub.py

```diff
@@ -166,8 +168,21 @@ def __init__(self,
     self._duration = duration
     # A signal for when to stop the metronome.
     self._stop_time = stop_time
+    self._qpb = qpb
+    self._count = 0
```

Comment:
count doesn't need to be an instance variable since it's only used inside of run